examples/curations.yml

```diff
@@ -75,3 +75,9 @@
   curations:
     comment: "Scan the source artifact, because the VCS revision and path are hard to figure out."
     source_code_origins: [ARTIFACT]
+
+- id: "Maven:androidx.collection:collection:"
+  curations:
+    comment: "Specify the platform for use from the policy rules."
```

Comment:
"Specify the platform to be used in the policy rules." / "Specify the platform used within the policy rules."

Reply:
I liked the "for use" wording, so I kept it, but changed it to "for use *within*" (but without "the", as no specific policy rules are meant here).